### PR TITLE
Update squads and names

### PR DESCRIPTION
**Pull request by diegopetrucci**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,15 +63,15 @@ At Babylon, we firmly believe that **transparency** is a core value that should
 By definition, members work on their respective squad, although they are free to work in different squads if the work load justifies it.
 
 
-| Squad Name                    | Members                                     | Availability | Public slack channel |
-|-------------------------------|---------------------------------------------| ------------ | -------------------- |
+| Squad Name                    | Members                                     | Availability | Public slack channel | What is the squad's area of competence? |
+|-------------------------------|---------------------------------------------| ------------ | -------------------- | --- |
 | Booking                       | Witold                                      |    1/1       | |
-| Condition Management          | Catarina, João, Daniel                      |    3/3       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
+| Condition Management          | Catarina, João, Daniel, Diego                      |    3/3       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
```
Comment (diegopetrucci) [inline README.md]:
```suggestion
| Condition Management          | Catarina, João, Daniel, Diego                      |    4/4       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) | 
```

File: README.md [hunk 2]
```diff
@@ -63,15 +63,15 @@ At Babylon, we firmly believe that **transparency** is a core value that should
...
+| Condition Management          | Catarina, João, Daniel, Diego                      |    3/3       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) | 
 | Consultation                  | Adrian, Chitra                              |    2/2       | |
 | Core Experience               | Sergey, Yuri                                |    2/2       | [#core-exp-squad](https://babylonhealth.slack.com/archives/CCSE8JLK0) |
 | Dev Experience                | Ilya, Olivier, David                        |    3/3       | [#mobile-tooling](https://babylonhealth.slack.com/messages/CL3QSC2NM) |
 | Early Feedback                | Danilo                                      |    1/1       | [#early_feedback](https://babylonhealth.slack.com/archives/CLGA76GG7) |
-| Enrolment and Integrity       | Diego, James                                |    2/2       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
+| Enrolment and Integrity       | James                                |    2/2       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
```
Comment (diegopetrucci) [inline README.md]:
```suggestion
| Enrolment and Integrity       | James                                |    1/1       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
```

File: README.md [hunk 3]
```diff
@@ -63,15 +63,15 @@ At Babylon, we firmly believe that **transparency** is a core value that should
 By definition, members work on their respective squad, although they are free to work in different squads if the work load justifies it.
 
 
-| Squad Name                    | Members                                     | Availability | Public slack channel |
-|-------------------------------|---------------------------------------------| ------------ | -------------------- |
+| Squad Name                    | Members                                     | Availability | Public slack channel | What is the squad's area of competence? |
+|-------------------------------|---------------------------------------------| ------------ | -------------------- | --- |
 | Booking                       | Witold                                      |    1/1       | |
-| Condition Management          | Catarina, João, Daniel                      |    3/3       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
+| Condition Management          | Catarina, João, Daniel, Diego                      |    4/4       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
```
Comment (AnnKatF) [inline README.md]: Can you change `Daniel` to `Dan` please?

File: README.md [hunk 4]
```diff
@@ -63,15 +63,15 @@ At Babylon, we firmly believe that **transparency** is a core value that should
...
 | Core Experience               | Sergey, Yuri                                |    2/2       | [#core-exp-squad](https://babylonhealth.slack.com/archives/CCSE8JLK0) |
 | Dev Experience                | Ilya, Olivier, David                        |    3/3       | [#mobile-tooling](https://babylonhealth.slack.com/messages/CL3QSC2NM) |
 | Early Feedback                | Danilo                                      |    1/1       | [#early_feedback](https://babylonhealth.slack.com/archives/CLGA76GG7) |
-| Enrolment and Integrity       | Diego, James                                |    2/2       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
+| Enrolment and Integrity       | James                                |    1/1       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
 | Healthcheck                   | Ben, Julien                                 |    2/3       | [#healthcheck_tribe](https://babylonhealth.slack.com/archives/C7995CX3R) |
 | Monitor                       | Anders, Joshua, Nicola                      |    3/3       | [#patient-metrics-team](https://babylonhealth.slack.com/archives/CE37S5W9Z) |
```
Comment (AnnKatF) [inline README.md]: Can you update this? Remove `Nicola` and add `Daniel` please?

File: README.md (hunk 3 above)
Comment (diegopetrucci) [inline README.md]:
```suggestion
| Condition Management          | Catarina, João, Dan S., Diego                      |    4/4       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) | 
```

File: README.md (hunk 4 above)
Comment (diegopetrucci) [inline README.md]:
```suggestion
| Monitor                       | Anders, Joshua, Daniel H.                      |    3/3       | [#patient-metrics-team](https://babylonhealth.slack.com/archives/CE37S5W9Z) |
```

**Comment by diegopetrucci** [comment]
@AnnKatF I've added their names + initial of last name since we seem to avoid adding last names

**Comment by diegopetrucci** [comment]
@anilputtabuddhi could you update your entry please? feel free to commit

**Comment by anilputtabuddhi** [comment]
> @anilputtabuddhi could you update your entry please? feel free to commit

Done in [4aae5ef](https://github.com/Babylonpartners/ios-playbook/pull/215/commits/4aae5ef968bf68ddcd6bcbaa5654bd1c79c43d58)

**Comment by MichalTKwiecien** [comment]
We have to remember that this repository is open source - some squads work on a features that are not released yet and maybe we shouldn't publicly say exactly what the works is about.

**Comment by scass91** [comment]
I'm not certain this is a necessary change? There's slack channels listed there where questions about what a person does can be asked.

**Review by j531 — COMMENTED** [review]
- Approve of squad updates
- Don't think the remit of each squad should be public. Risks leaking confidential information. Prefer an internal doc.

File: README.md [hunk 5]
```diff
@@ -63,25 +63,25 @@ At Babylon, we firmly believe that **transparency** is a core value that should
 By definition, members work on their respective squad, although they are free to work in different squads if the work load justifies it.
 
 
-| Squad Name                    | Members                                     | Availability | Public slack channel |
-|-------------------------------|---------------------------------------------| ------------ | -------------------- |
+| Squad Name                    | Members                                     | Availability | Public slack channel | What is the squad's area of competence? |
```
Comment (diegopetrucci) [inline README.md]:
```suggestion
| Squad Name                    | Members                                     | Availability | Public slack channel |
```

File: README.md [hunk 6]
```diff
@@ -63,25 +63,25 @@ At Babylon, we firmly believe that **transparency** is a core value that should
...
+|-------------------------------|---------------------------------------------| ------------ | -------------------- | --- |
```
Comment (diegopetrucci) [inline README.md]:
```suggestion
|-------------------------------|---------------------------------------------| ------------ | -------------------- |
```

**Comment by diegopetrucci** [comment]
@scass91 @j531 @MichalTKwiecien I've removed the extra column and just kept the squad changes

**Review by MichalTKwiecien — APPROVED** [review]

**Review by j531 — APPROVED** [review]

**Review by zopa-simon-cass — APPROVED** [review]
👍

**Review by danspndl — APPROVED** [review]

**Pull request merged**